plugins/SliceInfoPlugin/SliceInfo.py

```diff
@@ -119,6 +119,7 @@ def _onWriteStarted(self, output_device):
             }
             for container in global_container_stack.getContainers():
                 container_id = container.getId()
+                container_serialized = ""
```

Comment:
Additionally when serializing fails `container_serialized` is always filled with old data, but the container_id is always new. So at the end you'll set wrong data into a new container ID (keyword of the dict).